examples/3d/two_passes.rs

```diff
@@ -1,217 +0,0 @@
-use bevy::{
```

Comment:
I think this example is important to show rendering two cameras to the same screen.  It might be more interesting if they were different views.

Reply:
Fair point. I just added a new two_passes example that illustrates this.